Tasks/AzureIoTEdgeV2/Tests/package.json

```diff
@@ -17,6 +17,10 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "devDependencies": {
-    "@types/mocha": "^5.2.0"
+    "azure-pipelines-task-lib": "^3.0.6-preview.0",
```

Comment:
>preview [](http://example.com/codeflow?start=40&length=7)

I am a bit concerned to take a dependency on a preview package version. Is it possible to use a stable version instead?